src/apify_client/clients/resource_clients/request_queue.py

```diff
@@ -194,6 +220,19 @@ def get_request(self, request_id: str) -> dict | None:
 
         return None
 
+    def get_request_by_unique_key(self, request_unique_key: str) -> dict | None:
```

Comment:
I believe it's clear that it is the request's unique key (this applies to all occurrences).

```suggestion
    def get_request_by_unique_key(self, unique_key: str) -> dict | None:
```